docs/build_arm.md

```diff
@@ -1,14 +1,14 @@
 # Building Larq Compute Engine for ARM-based systems
-This page descibes how to build a Larq Compute Engine (LCE) inference binary
+This page descibes how to build Larq Compute Engine (LCE) applications
 for 32-bit, as well as 64-bit ARM-based systems. You can either build LCE
 natively on your ARM device or cross-compile it from a non-ARM machine.
 [Bazel](https://bazel.build/) is the primary build system for LCE, and it can
 be used to cross-compile a binary for ARM-based systems using a non-ARM host.
 To natively build on an ARM system itself we provide a script that uses the
 Makefile build system.
```

Comment:
I am strongly against using the word "application". Application has different meaning (see [here](https://en.wikipedia.org/wiki/Application_software)). The right word here is "binary" or "executable". Even the TF lite benchmark guide use the word "binary".

Reply:
I'll change it back to 'binary'.

I did have that particular meaning of the word application in mind, because the developers that write a program with LCE might want to use it to build an actual Android app, for example where LCE processes the camera input. In that case, application would be correct I think.